.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -5,6 +5,13 @@ body:
   - type: markdown
     attributes:
       value: "**Required Info:**"
+  - type: input
+    id: generative_ai
+    attributes:
+      label: "Generated by Generative AI"
+      description: "If this issue was generated using Generative AI, specify the model (e.g., GitHub Copilot v3.2)"
+    validations:
+      required: false

```

Comment:
this adds the optional field like below.

![image](https://github.com/user-attachments/assets/b22d6f0c-efa6-4170-97d0-5e65fe959133)